themes/openy_themes/openy_carnation/openy_carnation.theme

```diff
@@ -314,10 +314,15 @@ function openy_carnation_preprocess_page_logo(&$variables) {
  * Implements hook_preprocess_ENTITY_TYPE().
  */
 function openy_carnation_preprocess_paragraph(array &$variables) {
-  if (!Drupal::hasService('plugin.manager.home_branch_library')) {
+  if (empty($variables['paragraph'])) {
     return;
   }
-  if (empty($variables['paragraph'])) {
+
+  if ($variables['paragraph']->bundle() == 'gallery') {
+    $variables['#attached']['library'][] = 'openy_prgf_gallery/carnation.gallery_extras';
```

Comment:
@dchubar-fj Maybe better to attach library in twig template?

Reply:
@hamrant make sense, fixed